src/rdb/rdb_tx.c

```diff
@@ -1023,6 +1026,28 @@ rdb_tx_fetch(struct rdb_tx *tx, const rdb_path_t *kvs, enum rdb_probe_opc opc,
 	return rc;
 }
 
+/* Find the largest integer key in \a kvs
+ *
+ */
```

Comment:
Supposed to be a slightly better-formatted Doxygen, since this is an RDB API?

    /**
     * Find the largest largest integer key in \a kvs.
     *
     * \param[in] ...
     * ...
     */

Reply:
Missed adding this, have added documentation in latest commit.